src/Microsoft.Health.Abstractions/Microsoft.Health.Abstractions.csproj

```diff
@@ -2,6 +2,7 @@
   
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
+    <NeutralLanguage>en</NeutralLanguage>
```

Comment:
Per the FxCop error, neutral language required to be added. I chose English for now. Thoughts??

Reply:
I believe this is generally added via the `AssemblyInfo.cs` file in the various projects. And yes, it looks like we currently set it as `[assembly: NeutralResourcesLanguage("en-us")]`

Reply:
Thanks for the info. To be consistent with other projects, I created AssemblyInfo.cs for Abstractions project too.